python/tvm/relay/frontend/tflite.py

```diff
@@ -1370,11 +1378,7 @@ def convert_greater(self, op):
     def convert_squared_difference(self, op):
         """Convert TFLite SQUARED DIFFERENCE"""
         # Check if the input tensor is quantized, call QNN op
-        if self.is_quantized(op):
-            raise tvm.error.OpNotImplemented(
-                "TFlite quantized squared difference operator is not supported yet."
-            )
-        difference = self._convert_elemwise(_op.subtract, op)
+        difference = self._convert_elemwise(_op.subtract, op, dequantize=True)
```

Comment:
Shouldn't there be a check like the following?

```suggestion
        difference = self._convert_elemwise(_op.subtract, op, dequantize=self.is_quantized(op))
```